Library/Formula/racer.rb

```diff
@@ -0,0 +1,41 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
```

Comment:
When installing `racer` to `opt_prefix` and remove `key_only`, I got error
```sh
➜  Desktop  brew install racer
==> Downloading https://github.com/phildawes/racer/archive/v1.0.0.tar.gz
Already downloaded: /Library/Caches/Homebrew/racer-1.0.0.tar.gz
==> Downloading https://static.rust-lang.org/dist/rustc-1.3.0-src.tar.gz
Already downloaded: /Library/Caches/Homebrew/racer--rustsrc-1.3.0.tar.gz
==> cargo build --release
Error: An unexpected error occurred during the `brew link` step
The formula built, but is not symlinked into /usr/local
Directory not empty - /usr/local/opt/racer
Error: Directory not empty - /usr/local/opt/racer
```
Installing to `opt_prefix` create directory  `/usr/local/opt/racer` and install `racer` to it. And finally when link `/usr/local/Cellar/racer/1.0.0` to `opt_prefix`, it would be wrong because directory `/usr/local/opt/racer` not empty